src/bp/ui-studio/src/web/views/FlowBuilder/index.tsx

```diff
@@ -235,6 +236,7 @@ class FlowBuilder extends Component<Props, State> {
           readOnly={this.state.readOnly}
           mutexInfo={this.state.mutexInfo}
           permissions={panelPermissions}
+          onDeleteSelectedElements={() => this.diagram?.deleteSelectedElements()}
```

Comment:
No need for the anonymous function ` onDeleteSelectedElements={this.diagram?.deleteSelectedElements}`

Reply:
It actually doesn't work otherwise... 
I get this error
```
index.tsx:454 Uncaught TypeError: Cannot read property 'diagramEngine' of undefined
    at ./src/web/views/FlowBuilder/diagram/index.tsx.Diagram.deleteSelectedElements (index.tsx:454)
```
and I get if I add .bind(this)
```
index.tsx:454 Uncaught TypeError: Cannot read property 'getDiagramModel' of undefined
    at FlowBuilder../src/web/views/FlowBuilder/diagram/index.tsx.Diagram.deleteSelectedElements (index.tsx:454)
```
It seems like it binds to the elements before they're defined and having it in the anonymous function fixes it because they're defined by the time it's called